venom/rpc/reflect/stubs.py

```diff
@@ -13,7 +13,7 @@ class SchemaMessage(Message):
         'venom.rpc.reflect.openapi.SchemaMessage',
         json_name='additionalProperties'
     )
-    items = Field('venom.rpc.reflect.openapi.SchemaMessage')
+    items_ = Field('venom.rpc.reflect.openapi.SchemaMessage', json_name='items')
```

Comment:
Do tests pass with `name='items'`? If so, that would be better. Otherwise, keep it.